en/docs/security-announcements/security-advisories/2017/WSO2-2017-0184.md

```diff
@@ -31,10 +31,10 @@ The XXE attacks can affect any trusted system respective to the machine where th
 ### SOLUTION
 The recommended solution is to apply relevant security patch which fixes the XXE vulnerability in the WSO2 Carbon Commons component used in the listed products.
 
-Please see below for details on patching or updating the affected component.
+`See below for details on patching or updating the affected component.`
```

Comment:
Do we need to add these inside backticks (`` ` ``)

Reply:
Fixed in 27cf1e388a80e0db2fcda559634eacab8feb3e4a